frontend/apps/reader/modules/readerrolling.lua

```diff
@@ -1310,6 +1377,13 @@ Note that %1 (out of %2) xpaths from your bookmarks and highlights have been nor
     })
 end
 
+function ReaderRolling:onToggleHideNonlinear()
+    self.hide_nonlinear_flows = not self.hide_nonlinear_flows
+    self.ui.document:setHideNonlinearFlows(self.hide_nonlinear_flows)
+    self:onUpdatePos()
+    self:onChangeViewMode()
```

Comment:
A small comment about what you call each of these ones?

Reply:
A comment here or in the code? Basically, the document may change (due to the force pagebreaks), and the footer and toc markers need to be updated even if it doesn't change.

Reply:
In the code, so it survives :) 
And a comment about why onChangeViewMode() (what it does that you need, because one would wonder what view mode is changed here).